src/pages/settings/Profile/LoginField.js

```diff
@@ -86,7 +86,7 @@ class LoginField extends Component {
 
         return (
             <View style={styles.mb6}>
-                <Text style={styles.formLabel}>{this.props.label}</Text>
+                <Text style={[styles.formLabel, !this.props.login.validatedDate && styles.mb0]}>{this.props.label}</Text>
```

Comment:
FYI, phone number will always have 0 bottom-margin. I don't mind that honestly, looks better that way.

![image](https://user-images.githubusercontent.com/29673073/152345931-07d9f3a9-3178-402c-98dc-2c20aab05c77.png)